Plugins/Public/base_plugin/PlayerBase.cpp

```diff
@@ -28,18 +28,18 @@ PlayerBase::PlayerBase(uint client, const wstring &password, const wstring &the_
 	TranslateX(position, rotation, 1000);
 
 	// Create the default module and spawn space obj.
-	modules.push_back((Module*)new CoreModule(this));
+	modules.emplace_back((Module*)new CoreModule(this));
 
 	// Setup derived fields
 	SetupDefaults();
 
 	save_timer = rand() % 60;
 }
 
-PlayerBase::PlayerBase(const string &the_path)
+PlayerBase::PlayerBase(const string& the_path)
 	: path(the_path), base(0), money(0),
-	base_health(0), base_level(0), defense_mode(0), proxy_base(0), affiliation(0), siege_mode(false),
```

Comment:
why is siege_mode(false) being removed here and what does it have to do with crafting?

Reply:
Probably an honest mistake on my part, will re-add
Technically no functional change as bools initialize as false by default.